### PR TITLE
Update syntex to 0.38

### PR DESCRIPTION
**Pull request by dtolnay**

### CODE REVIEW

**Comment by oli-obk** [comment]
travis choked, not sure how to re-trigger it in this state.

changes lgtm

File: serde_codegen_internals/src/attr.rs [hunk 1]
```diff
@@ -517,33 +519,33 @@ struct Respanner<'a, 'b: 'a> {
...
                 TokenTree::Sequence(
                     self.new_span(span),
-                    tokenstream::SequenceRepetition {
-                        tts: self.fold_tts(seq.tts),
+                    Rc::new(tokenstream::SequenceRepetition {
+                        tts: self.fold_tts(&seq.tts),
                         separator: seq.separator.clone().map(|tok| self.fold_token(tok)),
-                        ..seq
-                    }
+                        ..**seq
```
Comment (oli-obk) [inline serde_codegen_internals/src/attr.rs]: I'm amazed this works... some interaction between copy types and deref?

**Comment by dtolnay** [comment]
> Jul 9, 17:44 UTC
> Identified - We've identified a set of builds which were causing our scheduling engine to throw an unexpected error. We've cancelled the problematic builds and are beginning to schedule jobs again.

> Jul 9, 17:34 UTC
> Investigating - We are investigating why jobs are not being scheduled on travis-ci.org

**Comment by dtolnay** [comment]
> Jul 9, 17:51 UTC
> We're processing jobs but with a significant backlog. We're scaling up a bunch more capacity for the container-based jobs to help catch up on the backlog faster.

**Comment by dtolnay** [comment]
Nice, they weren't kidding about extra capacity:

selection_002

**Comment by oli-obk** [comment]
someone should give these guys a medal

**Pull request merged**